kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/richmessaging/payment/PaymentActivity.java

```diff
@@ -49,7 +50,7 @@ protected void onCreate(Bundle savedInstanceState) {
         txnData = new HashMap<>();
         setWebViewClient();
 
-        String helpCenterUrl = getIntent().getStringExtra(KommunicateUI.KM_HELPCENTER_URL);
+        String helpCenterUrl = getIntent().getStringExtra(KmConstants.KM_HELPCENTER_URL);
```

Comment:
@reytum how is helpcenter related to Payment? (i see that this is an existing code, not related to this issue, but still curious to know, why it is placed in Payment?)

Reply:
During hotel booking bot development, this Activity was named as PaymentActivity, Since it does the web view reload work, we used it for all the web view related work later. Will have to rename it .